ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -384,45 +384,72 @@ if get_option('enable-openvino')
-      dependencies: nnstreamer_filter_mediapipe_deps,
-      include_directories: mediapipe_incdir,
-      install: true,
-      install_dir: filter_subplugin_install_dir
-    )
+  message('host system: ' + host_machine.system())
+  message('host cpu family: ' + host_machine.cpu_family())
+
+  if host_machine.system() == 'linux' and host_machine.cpu_family() == 'x86_64'
+    cmd = run_command('sh', '-c', 'echo $MEDIAPIPE_HOME')
+    MEDIAPIPE_HOME = cmd.stdout().strip()
+
+    mediapipe_env_exist = run_command('[', '-z', MEDIAPIPE_HOME, ']').returncode()
+    mediapipe_dir_exist = run_command('[', '-d', MEDIAPIPE_HOME, ']').returncode()
+
+    if mediapipe_env_exist == 1 and mediapipe_dir_exist == 0
+      message('MEDIAPIPE_HOME: ' + MEDIAPIPE_HOME)
+      mediapipe_incdir = include_directories(
+        MEDIAPIPE_HOME,
+        join_paths(MEDIAPIPE_HOME, 'bazel-bin'),
+        join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/eigen_archive'),
+        join_paths(MEDIAPIPE_HOME, 'bazel-mediapipe/external/com_google_absl'),
+      )
+
+      cmd = run_command('sh', '-c', meson.source_root() + '/tools/development/gen_mediapipe_libs.sh ' + nnstreamer_libdir)
+
+      mediapipe_internal_dep = cxx.find_library('mediapipe_internal', dirs: nnstreamer_libdir)
+      mediapipe_external_dep = cxx.find_library('mediapipe_external', dirs: nnstreamer_libdir)
+      opencv_core_dep = cc.find_library('opencv_core', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
+      opencv_video_dep = cc.find_library('opencv_video', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
+      opencv_imgcodecs_dep = cc.find_library('opencv_imgcodecs', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
+      opencv_imgproc_dep = cc.find_library('opencv_imgproc', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
+      opencv_calib3d_dep = cc.find_library('opencv_calib3d', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
+      opencv_highgui_dep = cc.find_library('opencv_highgui', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
+      opencv_videoio_dep = cc.find_library('opencv_videoio', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
+      opencv_features2d_dep = cc.find_library('opencv_features2d', dirs: join_paths(MEDIAPIPE_HOME, 'bazel-bin/_solib_k8/_U@linux_Uopencv_S_S_Copencv___Uexternal_Slinux_Uopencv_Slib_Sx86_U64-linux-gnu'))
```

Comment:
The joined path `bazel-...` is always same for all dependency. How about defining a path variable (for readability)?

Reply:
the variable has been created